hedera-node/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```

Comment:
The wrapper setup is now like this that there is only one Gradle Wrapper (scripts, jar, config) in the root of the repository. 
On the command line, you can still use that in one of the builds independently. For example:

```
> cd platform-sdk
> ../gradlew assemble
```

Unfortunately, if you only open the 'hedera-node' or 'platform-sdk' folder in IntelliJ IDEA, it does not find the wrapper in the parent folder. But if we add this here (only distributionUrl is enough), IDEA gets the Gradle version from here. 

Maybe this will be better in future IDEA versions. We could tweak this in a follow up, so that we have a Gradle update task, that updates the Gradle version on all the three `gradle-wrapper.properties` files ate once.

Reply:
I already removed the wrappers in the projects in this PR. It's only in the root now. And I think (?) calling it via `../gradlew` from one of the projects is fine. Then there is no need for additional scripts.

There is just a `gradle/wrapper/gradle-wrapper.properties` file in each project with the one line you see above for flawless IDEA integration. IDEA does not use the wrapper directly. It only reads the Gradle version from the `gradle/wrapper/gradle-wrapper.properties` file.